TACC/Core-CMS
# chore: load static not staticfiles templatetag

## Overview

Load `static` not `staticfiles` template tag, so #626 does not break.

## Related

- mimics https://github.com/TACC/Core-CMS/pull/625
- required by https://github.com/TACC/Core-CMS/pull/626

## Changes

- rename templatetag load in many templates

## Testing

1. Build TUP CMS using https://github.com/TACC/tup-ui/pull/225.
2. Load CMS in browser.
3. Verify no TemplateSyntaxError `'staticfiles' is not a registered tag library. Must be one of:`.

## UI

| Before | After |
| - | - |
| ![templatesyntaxerror](https://user-images.githubusercontent.com/62723358/235547759-d73e14b4-2423-475b-b1e5-1a126c7c51c2.png) | … |
